### DISCUSSION TITLE
Saved CSV Import on Status

### DISCUSSION THREAD

**Discussion by Serendibb** [burkybang/SuiteAdvanced-Feedback#199] [Ideas]
![image](https://github.com/user-attachments/assets/9126867b-4d12-4837-9aea-ab37914ee17a)
Lovely work on the View function. On this page, once I have an error, I will need to re run the Saved CSV Import again. Is it possible to show the Saved CSV Import used as a link directly on this page or line so it's easy to re-run it please?

**Comment by burkybang** [comment]
Thanks! I'm glad you like it.

That would be really useful. I'll keep this in mind, but my initial thought is I don't think it's plausible. There wouldn’t be any way to know programmatically which saved import was used. Also, the import might not have been saved.

**Comment by burkybang** [comment]
There will be a new feature similar to this in the next release!

**Comment by burkybang** [comment]
Released in version 31.2.0

Release notes: https://suiteadvanced.com/version-31_2_0
